chatkit-core/src/main/kotlin/com/pusher/chatkit/state/Actions.kt

```diff
@@ -14,6 +14,11 @@ internal data class JoinedRoom(
     val unreadCount: Int
 ) : Action()
 
+internal data class ReconnectJoinedRoom(
```

Comment:
as it's a special edge case I think it would be best to keep it below other main events, indeed how you put it in auxiliary state :)

Reply:
👍